common/api/bity.js

```diff
@@ -0,0 +1,82 @@
+import axios from 'axios';
+
+// https://stackoverflow.com/questions/9828684/how-to-get-all-arguments-of-a-callback-function
+export function combineAndUpper() {
+    let args = [];
+    let newString = '';
+    for (let i = 0; i < arguments.length; ++i) args[i] = arguments[i];
```

Comment:
Standard way to convert arguments to array. Easier to read than loop imo, also 1 line vs 2😛